data/src/main/scala/ch.epfl.scala.index.data/github/GithubReader.scala

```diff
@@ -99,8 +103,50 @@ object GithubReader {
     val repoTopicsPath = githubRepoTopicsPath(paths, github)
     val graphqlResult = read[GraphqlResult](
       Files.readAllLines(repoTopicsPath).toArray.mkString(""))
-    graphqlResult.data.repository.repositoryTopics.nodes.map { node =>
-      node.topic.name
+    graphqlResult.data.get.repository.get.repositoryTopics.get.nodes.get.map { node =>
+      node.topic.get.name.get
```

Comment:
Will do. Would replacing `get` with `flatMap` be better?

Ex. `graphqlResult.data.flatMap(_.repository.flatMap(_.repositoryTopics.flatMap(...`